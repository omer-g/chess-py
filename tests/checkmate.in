e2 e4
e7 e5
d1 h5
a7 a6
f1 c4
a6 a5
h5 f7
0
