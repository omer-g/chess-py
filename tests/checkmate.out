
        ##########################
        ######## chess-py ########
        ##########################

                    #
                   ###
                  ####
                 #### ##
                #### ####
                 ## ###
                  ####
               ##########
                  ####
                  ####
                  ####
                  ####
             ##############
             ##############

move: 'e2 e4' revert: 'r' exit: '0'

   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |bP|bP|bP|bP|bP|bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |  |  |  |  |  |  |
4 |  |  |  |  |  |  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|wP|wP|wP|wP|
1 |wR|wN|wB|wQ|wK|wB|wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |bP|bP|bP|bP|bP|bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |  |  |  |  |  |  |
4 |  |  |  |  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|wK|wB|wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |  |  |bP|  |  |  |
4 |  |  |  |  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|wQ|wK|wB|wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |bP|bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |  |  |  |  |bP|  |  |wQ|
4 |  |  |  |  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|  |wK|wB|wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |  |bP|bP|bP|  |bP|bP|bP|
6 |bP|  |  |  |  |  |  |  |
5 |  |  |  |  |bP|  |  |wQ|
4 |  |  |  |  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|  |wK|wB|wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |  |bP|bP|bP|  |bP|bP|bP|
6 |bP|  |  |  |  |  |  |  |
5 |  |  |  |  |bP|  |  |wQ|
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|  |wK|  |wN|wR|

enter move
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |  |bP|bP|bP|  |bP|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |bP|  |  |  |bP|  |  |wQ|
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|  |wK|  |wN|wR|

enter move
checkmate
   a  b  c  d  e  f  g  h  
8 |bR|bN|bB|bQ|bK|bB|bN|bR|
7 |  |bP|bP|bP|  |wQ|bP|bP|
6 |  |  |  |  |  |  |  |  |
5 |bP|  |  |  |bP|  |  |  |
4 |  |  |wB|  |wP|  |  |  |
3 |  |  |  |  |  |  |  |  |
2 |wP|wP|wP|wP|  |wP|wP|wP|
1 |wR|wN|wB|  |wK|  |wN|wR|

